requirements/pyproject.txt

```diff
@@ -10,6 +10,8 @@ pendulum==3.0.0
     # via pydantic-extra-types (pyproject.toml)
 phonenumbers==8.13.31
     # via pydantic-extra-types (pyproject.toml)
+bson==0.5.10
```

Comment:
why this is not added also to the other txt files in the requirements dir? such as requirements/pyproject.txt

Reply:
This was generated using [`pip-compile`](https://github.com/pydantic/pydantic-extra-types/blob/0b3191b4a3d55ecd19eae2b72bf015cd5135df1d/Makefile#L11), and we use it to separate dependencies that are for testing, linting, and also those that will be specified for running extra types.